recipes/datreant.core/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set version = "0.7.0" %}
+
+package:
+  name: datreant.core
+  version: "0.7.0"
+
+source:
+  git_url: https://github.com/datreant/datreant.core.git
+  git_tag: release-{{ version }}
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - python
+    - pip
+    - setuptools
+    - asciitree
+    - pathlib2
+    - scandir
+    - six
+    - fuzzywuzzy
+    - pytest
+    - datreant
+
+  run: &package_deps
```

Comment:
> run and test during the build are always in sync

Test dependencies are a superset of run dependencies, so you can drop `&package_deps` here and the `*package_deps` in `test/requires`, because this is already being done implicitly by `conda-build`

(TIL: yaml has anchors! that's really cool)